doc/api/inspector.md

```diff
@@ -52,6 +52,20 @@ parameter usage.
 
 Return the URL of the active inspector, or `undefined` if there is none.
 
+```js
+// Node.js process was called with --inspect flag.
+require('inspector').url()
```

Comment:
Linter is complaining about the missing semicolon here and on the other lines. That's easily correctable but it's also not liking the repeated `require('inspector')` which would be resolved by the further improvements talked about. Want to implement the console/shell display discussed instead? I'm pretty sure it will be Windows-compatible (and if it's not, that's something that can always be addressed in the text or by fixing it up in a subsequent PR).

Reply:
Yeah, definitely, I'll transpose this to console/shell style!